src/server/common/helpers/metrics/counter.js

```diff
@@ -0,0 +1,23 @@
+import {
+  createMetricsLogger,
+  Unit,
+  StorageResolution
+} from 'aws-embedded-metrics'
+import { config } from '~/src/config'
+import { createLogger } from '~/src/server/common/helpers/logging/logger'
+
+const logger = createLogger()
+
+const counter = async (metricName, value = 1) => {
+  if (!config.get('isProduction')) return
```

Comment:
Is the node `production` , not CDP env. So will still send metrics in dev, test , and perf test?

Reply:
Which we want, not just in prod. So ignore this

Reply:
my understanding is that this is 'node' production mode (i.e. not running npm test or npm run dev)

Reply:
https://nodejs.org/en/learn/getting-started/nodejs-the-difference-between-development-and-production